addons/l10n_dk/views/account_move_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_move_form_l10n_dk" model="ir.ui.view">
+        <field name="name">account.move.form.l10n_dk</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@name='journal_div']" position="after">
+                <field name="currency_rate_at_transaction" invisible="country_code != 'DK' or company_currency_id == currency_id"/>
```

Comment:
You forgot to rename your field here :nerd_face:

Reply:
:facepalm: